### PR TITLE
Add troubleshooting notes for Yarn/Git and adb server/client issues

### PR DESCRIPTION
**Pull request by vitvly**
This PR addes 2 troubleshooting notes for Git issues during `yarn install` and some adb info.

And also a new Makefile command: `make android-devices`.

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -42,3 +42,59 @@ _callImmediatesPass@http://localhost:8081/index.bundle?pla<…>
...
+## Cause
+`yarn.lock` is not updated to be in sync with `package.json`.
+
+## Solution
+Update yarn.lock file. In order to do this, perform the following steps on a clean `status-react` repo:
+```
+cd status-react
+cp mobile/js_files/package.json .
+yarn install
+cp yarn.lock mobile/js_files/
```
Comment (jakubgs) [inline TROUBLESHOOTING.md]: You could just use `ln` to symlink the files.

**Review by jakubgs — APPROVED** [review]
Nice.

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (vitvly) [reply TROUBLESHOOTING.md]: Yes, thanks, fixed.

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -42,3 +42,60 @@ _callImmediatesPass@http://localhost:8081/index.bundle?pla<…>
...
+
+## Cause
+`yarn.lock` is not updated to be in sync with `package.json`.
+
+## Solution
+Update yarn.lock file. In order to do this, perform the following steps on a clean `status-react` repo:
+```
+cd status-react
+ln -sf mobile/js_files/package.json .
+ln -sf mobile/js_files/yarn.lock .
```
Comment (yenda) [inline TROUBLESHOOTING.md:71]: you don't want to symlink this one since the point of `yarn install` is to recreate it. However maybe @PombeirP has a better solution, maybe `yarn add "git-url#tag"` works but I haven't tested it. That would avoid having all the unrelated changes to yarn.lock

Comment (jakubgs) [reply TROUBLESHOOTING.md:71]: Hmmm, going point actually.

Comment (vitvly) [reply TROUBLESHOOTING.md:71]: If symlinked, it's still updated correctly. And with a symlink one does not have to copy `yarn.lock` over again to `mobile/js_files`.

Comment (jakubgs) [reply TROUBLESHOOTING.md:71]: Yes, unless you want to re-create `yarn.lock` from scratch, but that's an extreme edge case, I doubt we'd want to do that, so whatever.

**Review by yenda — APPROVED** [review]
